File: javascript/easy/shuffle_string.js
// 1528. Shuffle String
// https://leetcode.com/problems/shuffle-string/

var restoreString = function(s, indices) {
    var ans = [];
    for (var i = 0; i < s.length; i++) {
        ans.push(s[indices.indexOf(i)])
    }
    
    return ans.join("");
};

File: python/easy/shuffle_string.py
# 1528. Shuffle String
# https://leetcode.com/problems/shuffle-string/

def restoreString(s, indices):
        answer = []
        for i in range(len(indices)):
            answer.append(s[indices.index(i)])
        
        return "".join(answer)

print(restoreString("codeleet", [4,5,6,7,0,2,1,3]))